db/seeds.rb

```diff
@@ -5,3 +5,33 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+books = [
+  {name: "Goodnight Moon", author: "Brittany", description: "A good book", ranking: 3},
+  {name: "Harry Potter", author: "Brittany", description: "A nice book", ranking: 4},
+  {name: "Curious George", author: "Brittany", description: "A cool book", ranking: 7},
+]
```

Comment:
If you want to seed your database with more data it's more convenient to use csv file